README.md

```diff
@@ -78,13 +78,13 @@ The default number of days is 7.
 
 Automatically close answered discussions as resolved
 
-This is enabled by default
+This is false by default
```

Comment:
The change we're making is to make the documentation accurate, so this doesn't need to change since we want `close-answered-discussions` to be true by default 🙂

Reply:
in that case, both are to be enabled by default. My bad! reverting the changes in documentation.

Reply:
The code for `CLOSE_LOCKED_DISCUSSIONS` will need to be updated to make it enabled by default to match the documentation in that case